xml/System/OperatingSystem.xml

```diff
@@ -603,7 +603,7 @@
         <param name="major">To be added.</param>
         <param name="minor">To be added.</param>
         <param name="build">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Check for the Mac Catalyst version (iOS version as presented in Apple documentation) with a ≤ version comparison. Used to guard APIs that were added in the given Mac Catalyst release.</summary>
         <returns>To be added.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the Mac Catalyst version is greater or equal than the specified version comparison; <see langword="false" /> otherwise.</returns>
```